packages/grafana-ui/src/components/CustomScrollbar/CustomScrollbar.tsx

```diff
@@ -1,8 +1,58 @@
 import React, { Component } from 'react';
 import isNil from 'lodash/isNil';
 import classNames from 'classnames';
+import { css } from 'emotion';
 import Scrollbars from 'react-custom-scrollbars';
 
+const getStyles = (page?: boolean) => {
```

Comment:
Can you move this to the bottom of the file, we usually move the getStyles to the bottom

Reply:
Alright 👍